pkg/health/etcdmember/builder_test.go

```diff
@@ -98,54 +95,90 @@ var _ = Describe("Builder", func() {
 				conditions := builder.Build()
 
 				Expect(conditions).To(ConsistOf(
-					oldMembers["1"],
-					oldMembers["2"],
```

Comment:
Is it intentional that the unmodified members should be not present in the build result?

Reply:
The members that are part of the result slice are taken into account, see [here](https://github.com/gardener/etcd-druid/blob/550a0b6a0f132419190043970fd34ae1a3df53ba/pkg/health/etcdmember/builder_test.go#L86). The rational behind this is that there will be one result per `Lease`. So when the cluster is scaled down from 3 -> 1, druid removes two member `Leases` and the health check will overwrite the members array with now only having one member because there is only one `Lease` left.

Reply:
Understood. Then it is intentional that the merge case is now no longer applicable, right? I.e. the builder always return as many members in the result slice as there are `leases`.

Reply:
Yes, it only needs the old members in order to set the `LastTransitionTime`.